securing_apps/topics.adoc

```diff
@@ -53,6 +53,8 @@ include::topics/oidc/javascript-adapter.adoc[]
 
```

Comment:
Hi @matthewhelmke the PR needs a rebase, but here comes few comments.

Reply:
Will rebase before the next commit.

Reply:
So, my rebase brought in a TON of other commits.